client.go

```diff
@@ -135,39 +135,70 @@ func (c *client) Walk(ctx context.Context, fid Fid, newfid Fid, names ...string)
 }
 
 func (c *client) Read(ctx context.Context, fid Fid, p []byte, offset int64) (n int, err error) {
-	resp, err := c.transport.send(ctx, MessageTread{
-		Fid:    fid,
-		Offset: uint64(offset),
-		Count:  uint32(len(p)),
-	})
-	if err != nil {
-		return 0, err
-	}
-
-	rread, ok := resp.(MessageRread)
-	if !ok {
-		return 0, ErrUnexpectedMsg
+	// size[4] Rread tag[2] count[4] data[count]
+	const rreadMessageEnvelopeSize int = 4 + 1 + 2 + 4
```

Comment:
This calculation should be a part of the message type. Please modify this correctly in codec.